relay-general/src/pii/processor.rs

```diff
@@ -16,8 +16,10 @@ use crate::protocol::{AsPair, IpAddr, NativeImagePath, PairList, User};
 use crate::types::{Meta, ProcessingAction, ProcessingResult, Remark, RemarkType};
 
 lazy_static! {
-    static ref NULL_SPLIT_RE: Regex = #[allow(clippy::trivial_regex)]
-    Regex::new("\x00").unwrap();
+    static ref NULL_SPLIT_RE: Regex = {
```

Comment:
can we add a code comment here if we're going to merge this

Reply:
This is even invalid syntax in Rust. Without `lazy_static`, this would not compile as statement and expression attributes aren't stable.